Tests/scripts/validate_premium_packs.py

```diff
@@ -0,0 +1,210 @@
+    """
+    request_data = \
+        {
+            'page': 0,
+            'size': 50,
+            'sort':
+                [{
+                    'field': 'updated',
+                    'asc': False
+                }],
+            'general': ["generalFieldPaid"]
+        }
+
+    logging.info(f'Getting premium packs from server {client.api_client.configuration.host}:\n')
+
+    # make the pack installation request
+    response_data, status_code, _ = demisto_client.generic_request_func(client,
+                                                                        path='/contentpacks/marketplace/search',
+                                                                        method='POST',
+                                                                        body=request_data,
+                                                                        accept='application/json',
+                                                                        _request_timeout=request_timeout)
+
+    if status_code == 200:
+        logging.debug(f'Got response data {response_data}')
+        response = ast.literal_eval(response_data)
+        logging.debug(f'Response dict is {response}')
+        logging.info('Got premium packs from server.')
+        return response["packs"]
+
+    result_object = ast.literal_eval(response_data)
+    message = result_object.get('message', '')
+    logging.error(f'Failed to retrieve premium packs - with status code {status_code}\n{message}\n')
+    return None
+
+
+def verify_server_paid_packs_by_index(server_paid_packs, index_data_packs):
+    """Compare two pack dictionaries and assert id's and prices are identical.
+
+    Raise AssertionError if the lists differ.
+
+    Args:
+        server_paid_packs: Dictionary of packs to check.
+        index_data_packs: Dictionary of packs to compare to.
+    """
+    # Sorting both lists by id
+    sorted_server_packs = sorted(server_paid_packs, key=lambda i: i['id'])
+    sorted_index_packs = sorted(index_data_packs, key=lambda i: i['id'])
+
+    # Checking lists are the same.
+    for (server_pack, index_pack) in zip(sorted_server_packs, sorted_index_packs):
+        assert server_pack["id"] == index_pack["id"]
+        assert server_pack["price"] == index_pack["price"]
+        logging.success(f'Pack: {server_pack["id"]} is valid.')
+
+
+def check_commit_in_master_history(index_commit_hash, master_history_path):
+    """Assert commit hash is in master history.
+
+    Args:
+        index_commit_hash: commit hash
+        master_history_path: path to a file with all the master's commit hash history separated by \n
+
+    Returns: None
+    """
+
+    with open(master_history_path, 'r') as master_history_file:
+        master_history = master_history_file.read()
+        master_commits = master_history.split('\n')
+
+    assert index_commit_hash in master_commits, f'Commit hash {index_commit_hash} is not in master history'
```

Comment:
again let's refrain from using assert - we should simply gather all the errors and print them in a report in the end.